### ISSUE TITLE
'attempt to subtract with overflow', src/display.rs:322:13

### ISSUE THREAD

**Issue by DmitriK** [bootandy/dust#162]
Stumbled upon this when trying to run 'dust -f' on my music directory and ended up with this in release build:
```
memory allocation of 18446744073709551614 bytes failed
fish: Job 1, 'dust -f' terminated by signal SIGABRT (Abort)
```
Re-ran dust as a debug build and got the error in the title. I haven't dug further, but I'm guessing it's due to some rather long path/filenames that I have that's hitting this.

**Comment by bootandy** [comment]
Thanks. 

I've been able to make it fail with the '-f' flag on one of my directories so I can reproduce it. I'll investigate and fix.

**Comment by bootandy** [comment]
Seems to be a bug introduced with the v0.6.0. Installing the old version and it works: `cargo install du-dust --version 0.5.4                   `.

I'll look into it.

**Comment by bootandy** [comment]
Found the bug. Thanks for the report. It'll be fixed with v 0.6.2
